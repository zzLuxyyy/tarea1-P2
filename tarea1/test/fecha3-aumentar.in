crearFecha 15/5/2024
imprimirFecha
aumentarDias 25
liberarFecha
Fin