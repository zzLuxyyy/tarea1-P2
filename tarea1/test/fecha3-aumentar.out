1>2>15/5/2024
3>La nueva fecha aplazada 25 dias es: 9/6/2024
4>5>Fin.
